.github/workflows/build.yml

```diff
@@ -3,15 +3,15 @@ name: BUILD PMM DOCS (2.x)
 on:
   push:
     branches:
-      - main
+      - separate-jobs
```

Comment:
Is this branch long-lived? Cause if you delete it, the workflow will never run.

Reply:
Right, it was just for testing, I intended fixing it after merge